src/screw_execution.cpp

```diff
@@ -0,0 +1,70 @@
+#include <affordance_primitives/screw_execution.hpp>
+#include <affordance_primitives/affordance_utils.hpp>
+#include <affordance_primitives/screw_axis.hpp>
+
+namespace affordance_primitives
+{
+APScrewExecutor::APScrewExecutor() : tfBuffer_(), tfListener_(tfBuffer_)
+{
+}
+
+APScrewExecutor::APScrewExecutor(ros::NodeHandle& nh, const std::string& server_name_)
+  : tfBuffer_(), tfListener_(tfBuffer_)
```

Comment:
Same here with the first `tfBuffer_( )` in the initializer list...  unless I'm missing something?